plugins/modules/system/sap_task_list_execute.py

```diff
@@ -0,0 +1,439 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sap_task_list_execute
+short_description: Perform SAP Task list execution.
+version_added: "3.5.0"
+description:
+  - The C(sap_task_list_execute) module depends on C(pyrfc) Python library (version 2.4.0 and upwards).
+    Depending on distribution you are using, you may need to install additional packages to
+    have these available.
+  - Tasks in the task list which requires manual activities will be confirmed automatically.
+  - This module will use the RFC Package C(STC_TM_API).
+
+options:
+  state:
+    description:
+      - If the state C(available_params) is provided the possible parameters are shown for each task.
+        The returned values are in the necessary format for C(task_parameters).
+      - The full list of parameters is exported to stdout in json format.
+    choices: ['available_params', 'present']
+    required: false
+    default: present
+    type: str
+  conn_username:
+    description: The required username for the SAP system.
+    required: true
+    type: str
+  conn_password:
+    description: The required password for the SAP system.
+    required: true
+    type: str
+  host:
+    description: The required host for the SAP system. Can be either an FQDN or IP Address.
+    required: true
+    type: str
+  sysnr:
+    description:
+      - The system number of the SAP system.
+      - This value must be single quoted. Otherwise ansible treated this like a number and replace leading zeros.
+      - Defaults to C('01')
+    required: false
+    default: '00'
+    type: str
+  client:
+    description:
+      - The client number to connect to.
+      - This value must be single quoted. Otherwise ansible treated this like a number and replace leading zeros.
+      - Defaults to C('001')
+    default : '000'
+    required: false
+    type: str
+  task_to_execute:
+    description: The task list which will be executed.
+    required: true
+    type: str
+  task_parameters:
+    description:
+      - The tasks and the parameters for execution.
+      - If the task list do not need any parameters. This could be empty.
+      - The list values must have curly brackets to declare a dictonary
+        and contain the following keys in uppercase C(TASKNAME), C(FIELDNAME), C(VALUE).
+        See examples.
+      - If only specific tasks from the task list should be executed.
+        The tasks even when no parameter is needed must be provided.
+        Alongside with the module parameter C(task_skip=true).
+    required: false
+    type: list
+    elements: dict
+    suboptions:
+      TASKNAME:
+        description: The name of the task in the task list.
+        type: str
+        required: true
+      FIELDNAME:
+        description: The name of the field of the task.
+        type: str
+        required: false
+      VALUE:
+        description: The value which have to be set.
+        type: str
```

Comment:
Just double checking: strings only? There is the `raw` type, if you need to leave it unspecified.

Reply:
i changed it :-) would be better